### PR TITLE
Added logo img

### PR DESCRIPTION
**Pull request by tlbanken**
@vinamartin @Kjames5269

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = {
           }
         },
         {
-          test: /\.(gif|png|jpe?g|svg)$/i,
```
Comment (vinamartin) [inline webpack.config.js:27]: ❓ I know we talked about it, but why did we get rid of svg again?

**Review by vinamartin — APPROVED** [review]
Looks good to me after my comment is addressed. ✅

**Comment by vinamartin** [comment]
Before this is merged, someone should "hero" this PR- in other words, someone should pull it down and build it locally to make sure it works as expected.

**Review by Kjames5269 — APPROVED** [review]

**Comment by Kjames5269** [comment]
Closing this PR as the repository has relocated to [Connexta/Eve](https://github.com/connexta/eve)

**Pull request closed without merging**